src/components/application_manager/include/application_manager/commands/command_impl.h

```diff
@@ -49,6 +51,20 @@ struct CommandParametersPermissions {
   RPCParams allowed_params;
   RPCParams disallowed_params;
   RPCParams undefined_params;
+
+  bool IsDisallowedParamsIncluded(const RPCParams& parameters) {
+    return std::includes(disallowed_params.begin(),
+                         disallowed_params.end(),
+                         parameters.begin(),
+                         parameters.end());
+  }
+
+  bool IsUndefindedParamsIncluded(const RPCParams& parameters) {
```

Comment:
@ZhdanovP What does it mean `undefinded`? May be `unfounded` or `undefined` , depends on context

Reply:
@AByzhynar please take a look a84499c9e385732690dfeca562d496cac9b3d483